HGustavs/LenaSYS
# G3 2020#8865 'Show more' button is hidden

To test this:

- Go to a course or a course-version that doesn't have more than 3 announcements 
- The 'show more' button should be hidden
- Then change back to a course or course-version that has more than 3 announcements the 'show more' button should be displayed.
